frontend/src/generate-static-html.js

```diff
@@ -1,24 +1,42 @@
 
```

Comment:
I figured out why this gets shipped to the web. Because it calls commonBrowserify, it gets put into the static/app directory. I wasn't expecting that. I'll have to change it to not call commonBrowserify.